Port over com.ibm.ws.logging.hpel_fat tests

Yushan-Lin opened the pull request:
Referring to https://github.ibm.com/was-liberty/WS-CD-Open/issues/20452

Yushan-Lin commented:
#build

Yushan-Lin commented:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_KztqMHjWEeqVMooXUPL4-g

Yushan-Lin commented:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_Qmh4MHjoEeqVMooXUPL4-g

fmhwong commented on dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/HPELDataDirFalsePositiveWarningTest.java:
```diff
@@ -0,0 +1,118 @@
+//%Z% %I% %W% %G% %U% [%H% %T%]
```
Missing copyright statement

fmhwong commented on dev/com.ibm.ws.logging.hpel_fat/test-applications/HpelFat/src/com/ibm/ws/logging/hpel/servlet/MyCustomHandler.java:
```diff
@@ -0,0 +1,11 @@
+package com.ibm.ws.logging.hpel.servlet;
```
Missing copyright statement

fmhwong commented:
I think most of the files were ported from WS-CD.  Can you describe what you have changed in the porting?

Yushan-Lin commented:
@fmhwong main thing that was changed was I removed `dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/fat/hpel/setup/HpelSetup.java` class and updated the tests so nothing is dependent on it anymore.

Yushan-Lin commented:
#run-libby-bot

Yushan-Lin commented:
#run-libby-bot

Yushan-Lin commented:
#run-libby-bot

Yushan-Lin commented:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_S6fj0IDIEeqspO8IkKlEbg
